aten/src/ATen/core/type.cpp

```diff
@@ -1385,11 +1409,39 @@ torch::jit::Function& ClassType::getMethod(const std::string& name) const {
   return *method;
 }
 
+std::vector<torch::jit::Function*> ClassType::findOverloadedMethod(const std::string& name) const {
+  if (overloaded_methods_.find(name) != overloaded_methods_.end()) {
+    return overloaded_methods_.find(name)->second;
+  }
+  return std::vector<torch::jit::Function*>();
+}
+
+torch::jit::Function* ClassType::getMangledOverloadedMethod(
+    const std::string& name) const {
+  if (mangled_to_function_.find(name) != mangled_to_function_.end()) {
+    return mangled_to_function_.find(name)->second;
+  }
+  return nullptr;
+}
+
+// std::vector<torch::jit::Function*> ClassType::getOverloadedMethod(const std::string& name) const {
```

Comment:
similar to getMethod, but we don't need it. It is because getMethod returns function reference and it is not possible to return the vector of function reference in C++